scripts/autoblog.js

```diff
@@ -0,0 +1,143 @@
+post2.setAttribute('class', 'post')
+
+//create image w/ src images/blog-2.jpg
+let img02 = document.createElement('img');
+img02.setAttribute('src', 'images/blog-2.jpg')
+img02.setAttribute('alt', 'four well-behaved dogs sit watching their dog walker enjoying the grand canyon')
+img02.setAttribute('width', 275)
+img02.setAttribute('height', 260)
+
+//create div (blog text container)
+let blogCopyContainer2 = document.createElement('div');
+
+//create blog entry title
+let entryTitle2 = document.createElement('h4');
+entryTitle2.textContent = 'How To Walk Multiple Dogs';
+
+//create blog body copy
+let entryCopy2 = document.createElement('p');
+entryCopy2.textContent = 'Iduciendisite quo magnatem iuntum quid quaest ea am, tenderumet adis dolenem quidustrum fuga. Faceaquae estioria derum recuptatur, cum volore, undipsa doloreium hillupta aut es ut alitatuscit ommossum haritatur arum qui of cae videbiti corporeium faccull oribus es quidignis ipietus explam sus am aut amet ant fugiatum, utem non reptat. Uptiusd andesci qui nem aut vendion ectur? Debis que explaut laborenia que doluptur, con et labor abor sant poreperum dio quat que doluptatur aut voluptassim quisciatquam ea ad qui con nes cus esere dolut hicto teni solutenis alit ulparume nonseca estorer spernam voluptassim quisciatquam ea ad qui con nes cus esere dolut hicto teni solutenis alit ulparume nonseca estorer spernam.';
+
+//ENTRY 3
+
+//create article w/ class 'post'
+let post3 = document.createElement('article');
+post3.setAttribute('class', 'post')
+
+//create image w/ src images/blog-3.jpg
+let img03 = document.createElement('img');
+img03.setAttribute('src', 'images/blog-3.jpg')
+img03.setAttribute('alt', 'a dog and its owner play fetch at sunset')
+img03.setAttribute('width', 275)
+img03.setAttribute('height', 260)
+
+//create div (blog text container)
+let blogCopyContainer3 = document.createElement('div');
+
+//create blog entry title
+let entryTitle3 = document.createElement('h4');
+entryTitle3.textContent = 'Teach Your Dog To Fetch!';
+
+//create blog body copy
+let entryCopy3 = document.createElement('p');
+entryCopy3.textContent = 'Iduciendisite quo magnatem iuntum quid quaest ea am, tenderumet adis dolenem quidustrum fuga. Faceaquae estioria derum recuptatur, cum volore, undipsa doloreium hillupta aut es ut alitatuscit ommossum haritatur arum qui of cae videbiti corporeium faccull oribus es quidignis ipietus explam sus am aut amet ant fugiatum, utem non reptat. Uptiusd andesci qui nem aut vendion ectur? Debis que explaut laborenia que doluptur, con et labor abor sant poreperum dio quat que doluptatur aut voluptassim quisciatquam ea ad qui con nes cus esere dolut hicto teni solutenis alit ulparume nonseca estorer spernam voluptassim quisciatquam ea ad qui con nes cus esere dolut hicto teni solutenis alit ulparume nonseca estorer spernam.';
+//Blog Entries END
+
+// Blog objects. 1 object per blog post.
+const blogObject1 = {
+  post: post1,
```

Comment:
A simpler method of doing this would be to put the `text` or `img url` directly into your array of objects ( see example below ). Then in your `for()` loop you can create / append the elements directly. This would cut down on the length of your code quite a bit.

```
const blogObjects = [
    { post: "sdfsdfsdfsdf", image: "img/url/here", title: "title"}
    { post: "sdfsdfsdfsdf", image: "img/url/here", title: "title"}
    { post: "sdfsdfsdfsdf", image: "img/url/here", title: "title"}
];
```

Reply:
Interesting! I think I understand the first part of your comment. So, make an array for each blog post and make an object that contains all arrays? 
With this way, I'm unsure of how to write the array. How would all the classes and attributes get assigned? I've attached a codepen link to an excerpt of my JS if that helps. Thanks! -S

https://codepen.io/SteveCarlton/pen/mdEdpoe?editors=0010